lib/formtastic_customizations.rb

```diff
@@ -17,9 +17,15 @@ def hint_html
           
           output = "".html_safe
           output << hint_html_without_customizations
-                    
+
+          current_person = template.instance_variable_get(:current_user).try(:person_type)
```

Comment:
Did you try just calling `@current_user` directly? Maybe it inherits the context automatically.